pycircleci/api.py

```diff
@@ -336,6 +336,24 @@ def cancel_build(self, username, project, build_num, vcs_type=GITHUB):
         resp = self._request(POST, endpoint)
         return resp
 
+    def get_pipelines(self, username, project, vcs_type=GITHUB):
+        """Get all pipelines configured for project.
+
+        :param username: Org or user name.
+        :param project: Repo name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Endpoint:
+            GET: ``/project/:vcs-type/:username/:project/pipeline``
```

Comment:
is it `pipeline` or `pipelines`?

According to  https://circleci.com/docs/2.0/api-intro/#new-endpoints, there is:
```
GET /project/:project_slug/pipelines/[:filter] 
```
and
```
POST /project/:project_slug/pipeline
```
Notice the singular in POST, and plural in GET.

Which one did you mean in this PR?

Reply:
Looks like the info in the link above is wrong. it's indeed `GET /project/:project_slug/pipeline` (singular) 🤷‍♂️ 
This one has it right: https://circleci.com/docs/api/v2/#get-all-pipelines